src/components/ResultPanel/index.tsx

```diff
@@ -4,7 +4,7 @@ import {CheckIcon} from '@Components/Icons';
 
 type ResultPanelProps = {
     iconClass?: string | null;
-    header: React.ReactNode;
+    header: string | React.ReactNode;
```

Comment:
`React.ReactNode` consist of a string as well in its type.

Reply:
Discussed.